.github/workflows/simorgh-unit-tests.yml

```diff
@@ -57,4 +57,5 @@ jobs:
         run: npm run test:unit
 
       - name: Report Code Climate Test Coverage
+        if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == 'bbc/simorgh' }} # Only run if PR originates from the Simorgh repo
```

Comment:
Does `||` not mean `or` in this context?

Don't understand the syntax here, but my assumption would be that if the `event_name == 'push'` the boolean will be true regardless of the `repo.full_name`

Reply:
Good spot, that is true. and this is so that when a psh event occurs (us merging to latest) we will run the CC test reporting

Reply:
Won't this also happen on push events on forks?  I really don't know enough about this so feel free to just merge, my uninformed assumption would just have expected the `||` to be `&&` 😄

Reply:
The actions are only triggered by push events when pushing to the latest branch https://github.com/bbc/simorgh/blob/latest/.github/workflows/simorgh-unit-tests.yml#L9

This means that the push even can only fire when merging to latest.

Reply:
ah cheers 👍 and that doesn't include the `latest` branch on forks?

Reply:
Nope. It does mean that when we merge a fork into latest this action will run which is expected as at that point the code is running within the simorgh repo so it will have access to the github secrets etc